### ISSUE TITLE
Unreal Tournament 2004 don't add foxWSFix

### ISSUE THREAD

**Issue by TheGreatestJannet** [luxtorpeda-dev/packages#674]
### Linux Distribution

Steam Flatpak

### Luxtorpeda Version

54

### Bug description

Luxtorpeda is adding the mod https://github.com/alexstrout/foxWSFix-UT2k4 to the game. This is causing issues when playing multiplayer as anticheat on servers check the game files and kick you if it detects the mod.

### Steps To Reproduce

1. Install the game with Luxtorpeda
2. Start the game
3. Join a server with anticheat
4. Get Kicked

### Expected Behavior

You shouldn't be kicked from the server.

### Additional Context

https://github.com/alexstrout/foxWSFix-UT2k4/issues/7

Maybe add an option to disable adding this mod.

**Comment by d10sfan** [comment]
@TheGreatestJannet Thanks for the issue.

The foxWSFix mod is added by doing the following

```
sed -i "s/InputClass=Class'Engine.PlayerInput'/InputClass=Class'foxWSFix.foxPlayerInput'/" linuxdata/System/User.ini
echo -e "\n\n[XGame.xPlayer]\nInputClass=Class'foxWSFix.foxPlayerInput'" >> linuxdata/System/User.ini
```

If you reverse that in the User.ini file (meaning that you set inputclass back to Engine.PlayerInput and remove the inputclass new line), is that enough to make that server work again?

**Comment by TheGreatestJannet** [comment]
Yes I removed the mod by reverting the changes in User.ini and deleting the mod files and the server stopped kicking me.

**Comment by d10sfan** [comment]
@TheGreatestJannet Did you need to delete the mod files, or was just changing the User.ini file enough?

**Comment by TheGreatestJannet** [comment]
I reverted the changes in User.ini but I was still getting kicked. Completely deleting the files stopped that. I'm not sure why as changing User.ini apparently should be enough to stop it loading.

**Comment by d10sfan** [comment]
@TheGreatestJannet Ah ok, thanks for the information. I'll see what I may be able to do to have options for it.

**Comment by JoshuaFern** [comment]
Unreal Tournament 99 also has a similar third party anti-cheat technology used by servers and might have a similar issue with compatibility depending on what tweaks Luxtorpeda adds to the game.

Might be something to look into.

**Comment by d10sfan** [comment]
Ut99 does not have any tweaks added onto it, other than using the latest 469 patch so it should not be affected but thanks for the reminder.

**Comment by d10sfan** [comment]
I added this in https://github.com/luxtorpeda-dev/packages/commit/126d0a5c79f22f7131786df8d9c15f6e41d82b77

Now when you launch the game, it'll use the engine chooser for with or without foxWSFix. This creates a linuxdata-standard folder for the one without the mod. When launching between each one, it will change the user.ini file in the user directory, to make sure the mod is either enabled or disabled.

Let me know if you run into any issues. I was able to use this to launch and join a mp server, and then go back and use the mod offline.
